Tasks/UsePythonVersion/task.json

```diff
@@ -34,6 +41,19 @@
             "required": true,
             "defaultValue": "true",
             "helpMarkDown": "Whether to prepend the retrieved Python version to the PATH environment variable to make it available in subsequent tasks or scripts without using the output variable."
+        },
+        {
+            "name": "architecture",
+            "type": "pickList",
+            "label": "Architecture",
+            "defaultValue": "x64",
+            "required": true,
+            "helpMarkDown": "The target architecture (x86, x64) of the Python interpeter.",
+            "groupName": "advanced",
+            "options": {
```

Comment:
I am not sure if this is appropriate here, but in most cases with options like this we have a "default" option that is selected automatically and allows the Task to make the most appropriate choice. I.e. choose the one that matches the OS arch.

Reply:
See a few lines above, the default is x64.  The agent's architecture won't be known at build definition time, but the hosted agents only have x64 Pythons.

Reply:
We have the "default" on several picklists (e.g. Java version) to make it just work on any agent but it does cause a lot of confusion. In this case not having "default" is better.

Reply:
Yes that is basically what I was thinking, but as Madhuri says, it can cause a lot of confusion. This is better.